services/quests/src/bot.tsx

```diff
@@ -0,0 +1,6 @@
+import * as Discord from 'discord.js';
```

Comment:
This seems like an odd place for this, since `quests` is run on the client. This should be part of the API server new quest publish route, so that we aren't sending a bunch of extra code to the client

Reply:
(this might also fix the tests failing)